src/http-spec.ts

```diff
@@ -72,13 +72,11 @@ export interface IHttpOperationResponse {
 export interface IHttpParam {
   name: string;
   style: HttpParamStyles;
+  content: IHttpContent;
```

Comment:
This is what we need — just keep in mind this will propagate in Graphite as a breaking change as well, since right now in OAS2 at least, we're pairing the single content with every single `produce` array element, which does not really make sense.